v21.1/show-backup.md

```diff
@@ -8,7 +8,7 @@ The `SHOW BACKUP` [statement](sql-statements.html) lists the contents of a backu
 
 ## Required privileges
 
-Only members of the `admin` role can run `SHOW BACKUP`. By default, the `root` user belongs to the `admin` role.
+`SHOW BACKUP` requires full read and write (including delete and overwrite) permissions to its target destination.
```

Comment:
Show backup only requires read permissions to its target destination. As a point above this, we should also add that `SHOW BACKUP` follows the semantics specified here - https://www.cockroachlabs.com/docs/stable/import.html#source-privileges

i.e some target destination require admin while other do not.